### PR TITLE
Update company-agreements.md

### PR DESCRIPTION
**Pull request by amynicol1985**

### CODE REVIEW

File: operations/operations/company-agreements.md [hunk 1]
```diff
@@ -6,11 +6,12 @@ Company agreements should only be signed by CEO, Ian Tien, after the approval of
...
 * If you are on-site and need to sign an NDA to enter a facility you may sign such an agreement 
 * The following agreements may be signed by VP Finance Aneal Vallurupalli 
-  * Non-clickthrough customer agreements up to $5,000 USD 
-  * Procurement contracts for software subscriptions, market events, marketing and advertising agency services up to $5,000 USD 
+  * Non-clickthrough customer agreements up to $50,000 USD 
+  * Procurement contracts for software subscriptions, market events, marketing and advertising agency services up to $50,000 USD 
   * Sponsored open source projects and paid simulations up to $600 USD
   * Unmodified Mattermost-authored Non-Disclosure Agreements 
   * Unmodified Mattermost-authored Partner Agreements 
+  * Investor quarterly reporting requests
```
Comment (it33) [inline operations/operations/company-agreements.md]: Thanks @amynicol1985 can we find Aneal's GitHub handle and mention him here? 

Propose we iterate on "Investor quarterly reporting requests" as a definition, as it's both really broad and really narrow.

**Comment by it33** [comment]
@jasonblais propose we have something in guidelines for approval on certain pages in handbook, e.g. CEO approval on any changes to signing authority (this PR).

**Comment by jasonblais** [comment]
@it33 Agree. I actually started a PR (still work in progress) to gather approved reviewers and permissions. Will continue to expand it: https://github.com/mattermost/mattermost-handbook/pull/54

(Also working with HR to pull people's GitHub handles so we can more easily tag people in handbook PRs as needed. Potentially via BambooHR)

**Comment by jasonblais** [comment]
@amynicol1985 has Aneal reviewed and signed off on the proposed changes here?

**Comment by amynicol1985** [comment]
> @amynicol1985 has Aneal reviewed and signed off on the proposed changes here?

@jasonblais - I don't believe so. How do I go about getting this done?

**Comment by jasonblais** [comment]
@amynicol1985 for now, you can ping Aneal directly on community.mattermost.com. I am working with HR on getting everyone's GitHub handles added to the Mattermost org, so you can easily request their reviews in pull requests, but we're not there yet

**Comment by it33** [comment]
Let's just remove "Investor quarterly reporting requests" and the other changes on amounts are fine. Let's do the MVC and merge this. 

The problem with "Investor quarterly reporting requests" is that it's both really specific to quarterly reports an yet really broad to "reporting requests" and we can iterate on that later. It's not a big deal, we've probably spent more time discussing the topic than the time it would have saved routing to Aneal vs. Ian, where as the change from $5K to $50K is a big change. 

This is a good first PR on agreements. Let's count it as a win and strong iteration. Let's get excited about iterating more.

**Comment by jasonblais** [comment]
Thanks Ian, removed, merging

**Review by jasonblais — APPROVED** [review]

**Pull request merged**